src/features/defi/providers/cosmos/components/CosmosManager/Overview/CosmosOverview.tsx

```diff
@@ -18,11 +18,12 @@ import { CircularProgress } from 'components/CircularProgress/CircularProgress'
 import { useBrowserRouter } from 'hooks/useBrowserRouter/useBrowserRouter'
 import { bn, bnOrZero } from 'lib/bignumber/bignumber'
 import { useGetAssetDescriptionQuery } from 'state/slices/assetsSlice/assetsSlice'
-import { makeTotalBondings } from 'state/slices/opportunitiesSlice/resolvers/cosmosSdk/utils'
+import { makeTotalCosmosSdkBondingsCryptoBaseUnit } from 'state/slices/opportunitiesSlice/resolvers/cosmosSdk/utils'
```

Comment:
Cosmos terminology because Cosmos SDK is one of the rare cases where the total crypto amount (delegations, rewards, undelegations) are in the same denom. It is the exception, not the rule and should not be applied to other opportunities